src/login/logind-dbus.c

```diff
@@ -2232,6 +2257,14 @@ static int method_schedule_shutdown(sd_bus_message *message, void *userdata, sd_
         if (r != 0)
                 return r;
 
+        if (!timestamp_is_set(elapse) && m->maintenance_window) {
+                r = calendar_spec_next_usec(m->maintenance_window, now(CLOCK_REALTIME), &elapse);
+                if (r < 0)
+                        return log_error_errno(r, "failed to parse shutdown time: %m");
+
+                log_info("scheduled %s at %s", type, FORMAT_TIMESTAMP(elapse));
```

Comment:
I think a log message here makes sense, but please uppercase the first char, and say explicitly that a maintanance shutdwn was requested, and what type